compiler/rustc_trait_selection/src/traits/error_reporting/suggestions.rs

```diff
@@ -1684,9 +1684,10 @@ impl<'tcx> TypeErrCtxtExt<'tcx> for TypeErrCtxt<'_, 'tcx> {
         true
     }
 
+    // return true if hit type references_error
```

Comment:
This comment is now inaccurate.

Reply:
yes. forgot to remove the old comment and add new comment.
will do it soon

Reply:
removed